lib/OpenSearch/Parameters/Search/Search.pm

```diff
@@ -1,582 +1,278 @@
-  metaclass   => 'MooseX::MetaDescription::Meta::Attribute',
-  description => {
-    encode_func => 'as_is',
-    type        => 'body',
-    required    => 0,
-  }
+  isa         => Int,
 );
 
 has 'seq_no_primary_term' => (
   is          => 'rw',
-  isa         => 'Bool',
-  metaclass   => 'MooseX::MetaDescription::Meta::Attribute',
-  description => {
-    encode_func => 'encode_bool',
-    type        => 'body',
-    required    => 0,
-  }
+  isa         => Bool,
 );
 
 has 'query' => (
   is          => 'rw',
-  isa         => 'HashRef',
-  metaclass   => 'MooseX::MetaDescription::Meta::Attribute',
-  description => {
-    encode_func => 'as_is',
-    type        => 'body',
-    required    => 0,
-  }
+  isa         => HashRef,
 );
 
 has 'min_score' => (
   is          => 'rw',
-  isa         => 'Int',
-  metaclass   => 'MooseX::MetaDescription::Meta::Attribute',
-  description => {
-    encode_func => 'as_is',
-    type        => 'body',
-    required    => 0,
-  }
+  isa         => Int,
 );
 
 has 'indices_boost' => (
   is          => 'rw',
-  isa         => 'ArrayRef[HashRef]',
-  metaclass   => 'MooseX::MetaDescription::Meta::Attribute',
-  description => {
-    encode_func => 'as_is',
-    type        => 'body',
-    required    => 0,
-  }
+  isa         => ArrayRef[HashRef],
 );
 
 has 'from' => (
   is          => 'rw',
-  isa         => 'Int',
-  metaclass   => 'MooseX::MetaDescription::Meta::Attribute',
-  description => {
-    encode_func => 'as_is',
-    type        => 'body',
-    required    => 0,
-  }
+  isa         => Int,
 );
 
 has 'explain' => (
   is          => 'rw',
-  isa         => 'Str',
-  metaclass   => 'MooseX::MetaDescription::Meta::Attribute',
-  description => {
-    encode_func => 'as_is',
-    type        => 'body',
-    required    => 0,
-  }
+  isa         => Str,
 );
 
 has 'fields' => (
   is          => 'rw',
-  isa         => 'ArrayRef',
-  metaclass   => 'MooseX::MetaDescription::Meta::Attribute',
-  description => {
-    encode_func => 'as_is',
-    type        => 'body',
-    required    => 0,
-  }
+  isa         => ArrayRef,
 );
 
 has 'docvalue_fields' => (
   is          => 'rw',
-  isa         => 'ArrayRef[HashRef]',
-  metaclass   => 'MooseX::MetaDescription::Meta::Attribute',
-  description => {
-    encode_func => 'as_is',
-    type        => 'body',
-    required    => 0,
-  }
+  isa         => ArrayRef[HashRef],
 );
 
 has 'aggs' => (
   is          => 'rw',
-  isa         => 'HashRef',
-  metaclass   => 'MooseX::MetaDescription::Meta::Attribute',
-  description => {
-    encode_func => 'as_is',
-    type        => 'body',
-    required    => 0,
-  }
+  isa         => HashRef,
 );
 
 has 'search_after' => (
   is          => 'rw',
-  isa         => 'ArrayRef',
-  metaclass   => 'MooseX::MetaDescription::Meta::Attribute',
-  description => {
-    encode_func => 'as_is',
-    type        => 'body',
-    required    => 0,
-  }
+  isa         => ArrayRef,
 );
 
 has 'scroll_id' => (
```

Comment:
`scroll_id` does not do anything here (only works in `/_search/scroll`). But something for another issue

Reply:
Yes. This is a leftover from the first version of the Perl-Module that i played with. But you're right. Should not be here